apps/expo/tailwind.config.ts

```diff
@@ -32,6 +32,9 @@ export default {
         'slate-900': "#1C1B1B",
         'dark-purple': "#48476D",
         'slate-200': "#CACACA",
+        'bb-dark-gray': '#272727',
+        'bb-dark-purple': '#48476D',
+        'bb-slate-100': '#1E1E1E'
```

Comment:
FYI: prefixing new tailwind colors here with "bb" for Barbell 8^)

Reply:
kkk